### PR TITLE
Some improvements of the SDKs

### PR DESCRIPTION
**Pull request by Kirbyrawr**
-Added Unity SDK to the list of SDKs in the reference/readme.md file.
-Changed C# SDK Github repository from QuantozTechnology to Elucidsoft in guides/get-started/readme.md, since this one is more supported.

### CODE REVIEW

File: guides/get-started/readme.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Using the Stellar network, you can build mobile wallets, banking tools, smart de
 
 The easiest way to install Horizon is by using [**stellar/quickstart** docker image](https://hub.docker.com/r/stellar/quickstart/).
 
-Stellar.org maintains [JavaScript](https://github.com/stellar/js-stellar-sdk), [Java](https://github.com/stellar/java-stellar-sdk), and [Go](https://github.com/stellar/go/tree/master/clients/horizon)-based SDKs for communicating with Horizon. There are also community-maintained SDKs for [Ruby](https://github.com/stellar/ruby-stellar-sdk), [Python](https://github.com/StellarCN/py-stellar-base), and [C#](https://github.com/QuantozTechnology/csharp-stellar-base).
+Stellar.org maintains [JavaScript](https://github.com/stellar/js-stellar-sdk), [Java](https://github.com/stellar/java-stellar-sdk), and [Go](https://github.com/stellar/go/tree/master/clients/horizon)-based SDKs for communicating with Horizon. There are also community-maintained SDKs for [Ruby](https://github.com/stellar/ruby-stellar-sdk), [Python](https://github.com/StellarCN/py-stellar-base), and [C#](https://github.com/elucidsoft/dotnet-stellar-sdk).
```
Comment (bartekn) [inline guides/get-started/readme.md]: Could you change the last sentence about community-maintained SDKs to the link to the list of SDKs in `reference/readme.md`?

Comment (Kirbyrawr) [reply guides/get-started/readme.md]: Sure will do when i arrive at home!

**Comment by bartekn** [comment]
I mean, instead of:
```
[Ruby](https://github.com/stellar/ruby-stellar-sdk), [Python](https://github.com/StellarCN/py-stellar-base), [C# .NET Core 2.0](https://github.com/elucidsoft/dotnetcore-stellar-sdk), [C#](https://github.com/QuantozTechnology/csharp-stellar-base), [C++](https://bitbucket.org/bnogal/stellarqore/wiki/Home), [Scala](https://github.com/Synesso/scala-stellar-sdk) and [Unity](https://github.com/Kirbyrawr/stellar-unity)
```
we should just link to `reference/readme.md` so we don't have to maintain this list in two places.

**Comment by Kirbyrawr** [comment]
I will take a look, thank you.

**Pull request closed without merging**